specs/protocol/isthmus/exec-engine.md

```diff
@@ -0,0 +1,45 @@
+# L2 Execution Engine
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Fees](#fees)
+  - [Configurable fees](#configurable-fees)
+    - [Configuring scalars](#configuring-scalars)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Fees
+
+New OP stack variants have different resource consumption patterns, and thus require a more flexible
+pricing model. To enable more customizable fee structures, Isthmus adds a new component to the fee
+calculation: the `ConfigurableFee`, which is parameterized by two scalars: the `configurableFeeScalar`
+and the `configurableFeeConstant`.
```

Comment:
I find the use of the prefix "configurable" a bit meaningless for this feature. Other fee parameters, like the `(blob)BaseFeeScalar`s are also "configurable". Maybe we use a prefix that better describes the reason for their introduction, like
* `OperatorFee`
* `operatorFeeScalar`
* `operatorFeeConstant`
or something similar that attaches more meaning to them? `fixedFee...` could also work.